rcon/automods/level_thresholds.py

```diff
@@ -69,7 +69,9 @@ def watch_state(self, team: str, squad_name: str):
                 pickle.dumps(watch_status)
             )
 
-    def on_connected(self, name: str, steam_id_64: str) -> PunitionsToApply:
+    def on_connected(
+        self, name: str, steam_id_64: str, detailed_player_info: GetDetailedPlayer
```

Comment:
Because detailed player info can fail, not sure if we want to support passing in a `None` `detailed_player_info` or an empty dict or something else.